pages/[displayStyle]/[[...page]].tsx

```diff
@@ -30,30 +32,21 @@ export default function HomePage(props: HomePageProps) {
     </Layout>
   );
 }
-
-export const REVALIDATION_TIME = 15 * 60; // 15 minutes
-const MAX_PAGES = 5;
-
 export const getStaticPaths = async () => {
   try {
     const prisma = await openConnection();
-
-    const [gridCursors, listCursors] = await Promise.all([
-      await getArticlesPaginationForGrid(prisma),
-      await getArticlesPaginationForList(prisma),
+    const [gridLastPage, listLastPage] = await Promise.all([
+      await getLastBlogPage(prisma),
+      await getLastArticlePage(prisma),
     ]);
-
+    const gridPages = getPagesArray(gridLastPage, MAX_PAGES);
+    const listPages = getPagesArray(listLastPage, MAX_PAGES);
     const paths = [
-      ...gridCursors
-        .slice(0, MAX_PAGES)
-        .map((cursor) => ({ params: { displayStyle: 'grid' as const, cursor } })),
-      ...listCursors
-        .slice(0, MAX_PAGES)
-        .map((cursor) => ({ params: { displayStyle: 'list' as const, cursor } })),
+      ...gridPages.map((page) => ({ params: { displayStyle: 'grid' as const, page: [page] } })),
```

Comment:
Czemu page jako tablica?
```suggestion
      ...gridPages.map((page) => ({ params: { displayStyle: 'grid' as const, page: page } })),
```

Reply:
w związku z obsługiwaniem również index w [[...page]]

Reply:
Ale nie musi być tablicą, nie? :P To jedna liczba.